### PR TITLE
add map_copy_read_only

### PR DESCRIPTION
**Pull request by zserik**
see also: https://github.com/danburkert/memmap-rs/pull/82

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -70,6 +83,7 @@ impl MmapOptions {
     /// # Ok(())
     /// # }
     /// ```
+    #[inline]
```
Comment (RazrFalcon) [inline src/lib.rs:86]: We don't need it.

File: src/lib.rs [hunk 2]
```diff
@@ -40,13 +40,26 @@ use std::usize;
...
 /// [`map_copy()`]: MmapOptions::map_copy()
+/// [`map_copy_read_only()`]: MmapOptions::map_copy_read_only()
 #[derive(Clone, Debug, Default)]
 pub struct MmapOptions {
     offset: u64,
     len: Option<usize>,
     stack: bool,
 }
 
+/// Wraps an `MmapInner` struct inside of a `Mmap` struct. Helper function.
```
Comment (RazrFalcon) [inline src/lib.rs:51]: Move any unrelated changes to a separate pull request.

**Review by RazrFalcon — CHANGES_REQUESTED** [review]

**Review by RazrFalcon — APPROVED** [review]

**Comment by RazrFalcon** [comment]
Can you force-push to restart the CI?

**Comment by zserik** [comment]
Done.

**Pull request merged**